public/assets/js/home.js

```diff
@@ -0,0 +1,91 @@
+
+function darkmode() {
+  var element = document.body;
```

Comment:
No sir, actually i have fixed this issue on the previous day only when i was working on preserving the dark mode functionality, but wasn't able to push since my github was down yesterday.